Manually add contacts to target groups (#52)
vivid-planet/comet-brevo-module: .changeset/dirty-balloons-prove.md

---
"@comet/brevo-admin": minor
"@comet/brevo-api": minor
---

> Contacts can now be manually assigned on the target group edit admin page additionally to the previously existing filter assignment

Reviewer: ```suggestion
Allow manually assigning contacts to a target group

This is in addition to the existing automatic assignment via filters.
```

Author: [a7a7b56](https://github.com/vivid-planet/comet-brevo-module/pull/52/commits/a7a7b56c3bfdf92dcd41474c1cfe73a14d2b535c)

---
"@comet/brevo-admin": minor
"@comet/brevo-api": minor
---

> Allow manually assigning contacts to a target group
> This is in addition to the existing automatic assignment via filters.

Reviewer: ```suggestion
Allow manually assigning contacts to a target group

This is in addition to the existing automatic assignment via filters.
```

The new line here is necessary to differentiate between title and description

Author: [19ddf00](https://github.com/vivid-planet/comet-brevo-module/pull/52/commits/19ddf00d7ed8457a250ae66a85a49849e0de536f)
